tree-sitter-stack-graphs/src/cli/analyze.rs

```diff
@@ -203,7 +209,9 @@ impl AnalyzeArgs {
             Err(LoadError::ParseErrors(parse_errors)) => {
                 let parse_error = map_parse_errors(source_path, &parse_errors, &source, "");
                 file_status.error("parsing failed")?;
-                eprintln!("{}", parse_error);
+                if !self.hide_error_details {
+                    println!("{}", parse_error);
```

Comment:
Do we want to print this on stderr?

Reply:
I am not sure, tbh. I have tried to find guidance on what should go on stderr vs stdout, but I didn't find much. It is not an error condition for `analyze`, in the sense that it will continue and complete without error (exit code). But if you have an idea of how to decide this, I'd love to hear it.